app/controllers/comments_controller.rb

```diff
@@ -47,7 +47,10 @@ def edit
   # POST /comments
   # POST /comments.json
   def create
-    raise if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
+    if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
+      skip_authorization
+      return head(:too_many_requests)
```

Comment:
It seems unnecessary to return a body here, but we can change this to `render json: {}, status: :too_many_requests` if you'd prefer to keep this more in line with other methods in this controller.

Reply:
I would indeed keep the body, it's also a `SHOULD` in the RFC: https://tools.ietf.org/html/rfc6585#section-4. In the future we might even want to tell the requester what's the limit, though we might want to discuss that first.

Reply:
Yup, good point regarding the RFC. I made it an empty body for now, should we include some sort of generic message like "Rate limit reached"?